packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx

```diff
@@ -1352,7 +1364,21 @@ export class FieldEditor {
                   ></fw-input>
                 </div>
               </span>
-              {toggle}
+              {isSlaEnabled &&
```

Comment:
@Elayaraman Do we need to have isSlaEnabled check here as this is for resolution_timer?

Reply:
No need for it here 
mistakenly added